packages/cli/src/worker.ts

```diff
@@ -29,28 +34,76 @@ function importHelperPlugin(babel: typeof import("@babel/core")) {
   };
 }
 
+const babelParser: typeof import("@babel/parser") = require(resolveFrom(
+  require.resolve("@babel/core"),
+  "@babel/parser"
+));
+
+const babelGenerator: typeof import("@babel/generator") = require(resolveFrom(
+  require.resolve("@babel/core"),
+  "@babel/generator"
+));
+
 export async function transformBabel(
   code: string,
   cwd: string,
   filename: string
 ) {
-  const babel = lazyRequire<typeof import("@babel/core")>();
-
-  return babel
-    .transformAsync(code, {
-      caller: {
-        name: "rollup-plugin-babel",
-        supportsStaticESM: true,
-        supportsDynamicImport: true,
+  const { generatorCacheKey,parseCacheKey, options } = resolveOptions({
+    caller: {
+      name: "rollup-plugin-babel",
```

Comment:
it probably would be a good idea to change this name - it usually doesn't really affect anyone, but might popup in the logs etc so it would be best if one would not think that this is using the original `rollup-plugin-babel` as one might report an issue against wrong repo or smth